docker/prod

```diff
@@ -20,6 +20,7 @@ ENV LANGUAGE en_US.UTF-8
 
 COPY requirements /app/requirements/
 RUN pip install --no-cache-dir --require-hashes --no-deps -r requirements/prod.txt
+RUN pip install psycopg2
```

Comment:
Are we going to use Postrgresql? If yes, let's cleanup the Mysql deps and fix the docker-compose too